.github/workflows/build.yml

```diff
@@ -0,0 +1,54 @@
+name: build Rbus
+on:
+  push:
+     branches: 
```

Comment:
Can you add both `main` and `rbus-2.0`

I guess you can pass `rbus-**` to build future branches like `2.1`, `2.2`, `3.0` etc.. But im not really sure of naming convention.

Please create a branch in your fork and run the test pl.